handlers/events.go

```diff
@@ -1,10 +1,33 @@
 package handlers
 
 import (
+	"github.com/alexdmtr/www/services/eventService"
+	"log"
 	"net/http"
 )
 
 func events(w http.ResponseWriter, r *http.Request) {
-	// TODO(andrei): Finish the events page.
-	templates["comingsoon"].ExecuteTemplate(w, "layout", nil)
+
+	var eventsContext struct {
+		EventGroup   *eventService.EventGroup
+		HaveRightNow bool
+		HaveUpcoming bool
+	}
+	eventGroup, err := eventService.GroupEvents()
+
+	if err != nil {
+		log.Println(err)
+		errorHandler(w, r, http.StatusServiceUnavailable)
+		return
+	}
```

Comment:
Very nice way of handling the error as well as displaying something to the user!

Reply:
Thanks!